<launch>

  <node pkg="network_planner"
        type="network_planner_node"
        name="network_planner_node"
        output="screen">
    <param name="sample_count"    value="5000"/>
    <param name="sample_variance" value="2.0"/>
    <param name="max_velocity"    value="1.0"/> <!-- m/s -->
  </node>

</launch>
